library/clinical_rules.php

```diff
@@ -2777,30 +2781,6 @@ function collect_database_label($label, $table)
     return $returnedLabel;
 }
 
-/**
```

Comment:
to help with the review on this, I just removed this function since it was not being used.